make/migrations/postgresql/0015_1.10.0_schema.up.sql

```diff
@@ -44,4 +44,17 @@ CREATE TABLE immutable_tag_rule
   creation_time timestamp default CURRENT_TIMESTAMP
 );
 
-ALTER TABLE robot ADD COLUMN visible boolean DEFAULT true NOT NULL;
\ No newline at end of file
+ALTER TABLE robot ADD COLUMN visible boolean DEFAULT true NOT NULL;
+
+/** Drop the unused vul related tables **/
```

Comment:
Is this change intentionally put in this PR?

Reply:
Yes, together with the deletions of DAO related files.